browser/browser.spec.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+describe('DOM Elements' ,() => {
+  let document;
+  beforeEach(async() => {
+    const dom = new JSDOM(`
+    <div class="container">
+      <div class="titles">
+        <h2 id="title">This is a title</h2>
+        <h4 class="subtitle">This is a subtitle</h4>
+      </div>
+      <div class="iterables">
+        <ul>
+          <li>Item 1</li>
+          <li>Item 2</li>
+          <li>Item 3</li>
+          <li>Item 4</li>
+        </ul>
+      </div>
+    </div>
+    `);
+    document = dom.window.document;
+  });
+
+  test('Should render the titles', () => {
+    // Getting element by tag
+    const title = document.querySelector('h2');
+
+    // Getting element by class
+    const subtitle = document.querySelector('.subtitle');
+    
+    // Getting element by Id
+    const titleById = document.querySelector('#title');
```

Comment:
Could we use `getElementById`?

Reply:
Yeah, but with `jsdom` is not possible. So, I avoid that part.
But, I could just mention it.